lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -505,6 +562,69 @@ static const std::string Bf16_to_Fp8E5M2(bool hasNativeFP) {
   return ret;
 }
 #endif
+
+// ROCM type conversion between fp8 and bf16
+#ifdef USE_ROCM
+
+static Value cvtBf16ToFp32(Location loc, 
+  ConversionPatternRewriter &rewriter, Value v) {
+  auto bf16x2VecTy = vec_ty(i16_ty, 2);
+  Value a0 = undef(bf16x2VecTy);
+  a0 = insert_element(bf16x2VecTy, a0, int_val(16,0), i32_val(0));
+  a0 = insert_element(bf16x2VecTy, a0, v, i32_val(1));
+  return bitcast(a0, i32_ty);
+}
+
+static Value cvtFp32ToBf16(Location loc, 
```

Comment:
This seems to truncate fp32 to bf16 instead of rounding it to bf16. There is already a function implementing the conversion from fp32 to bf16 in this file: convertFp32ToBf16

Reply:
Thanks, fixed.